### PR TITLE
fix(sync in home.component): fixed the stream code is home.component.ts

### LINKED ISSUE #2: Don't use tap to set properties, certainly if it can use subscribe.
https://github.com/thinktecture/ngeurope-2018-pwa/blob/1cf098182d6ba2d1ca5e480f0e95363bf3f92189/client/src/modules/app/components/home/home.component.ts#L67

On line 67 you use tap. You should do this in the subscription
   ```
 public sync(): void {
        fromPromise(this._todoService.getAll(true))
            .pipe(
                switchMap(items => this._syncService.sync(items)),
                switchMap(result => fromPromise(this._todoService.overwrite(result as Array<ITodoItem>)))
            ).subscribe(items => this.items = items));
    }
```

also, if you subscribe, you should unsubscribe, in ngOnDestroy

### PR DESCRIPTION
**Pull request by filipvh**
Closes #2

### CODE REVIEW

**Comment by christianliebel** [comment]
@filipvh Looks good. Could you place the import in one single line?

File: client/src/modules/app/components/home/home.component.ts [hunk 1]
```diff
@@ -5,7 +5,10 @@ import {SyncService} from '../../../shared/services/base/sync.service';
 import {TodoListComponent} from '../todoList/todoList.component';
 import {AppStateService} from '../../../shared/services/appState.service';
 import {Subscription} from 'rxjs/Subscription';
-import {switchMap, tap} from 'rxjs/operators';
+import {
+    switchMap,
+    take
+} from 'rxjs/operators';
```
Comment (christianliebel) [inline client/src/modules/app/components/home/home.component.ts]: One line please

**Review by christianliebel — CHANGES_REQUESTED** [review]

**Comment by chliebel** [comment]
Closes #2

File: client/src/modules/app/components/home/home.component.ts (hunk 1 above)
Comment (filipvh) [reply client/src/modules/app/components/home/home.component.ts]: np, will do, we do this actually to get less problems with merges

**Comment by filipvh** [comment]
@chliebel done ^.^

**Pull request merged**